e2e-tests/playwright/tests/functional/channels/settings/display/theme_accessibility.spec.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect} from '@playwright/test';
+
+import {test} from '@mattermost/playwright-lib';
```

Comment:
nit: the same `expect` is also available at `@mattermost/playwright-lib`

Reply:
Will do!

We have a similar issue in the web app code with React Testing Library exports that we re-export from our testing utilities. We might want to look into using the `no-restricted-imports` ESLint rule for both of those to push people to using the proper import